jobs/sportpursuit-python-developers-x2-roles.html

```diff
@@ -0,0 +1,70 @@
+---
+title: Python Developers (x2 roles)
+company: SportPursuit
+url: https://www.sportpursuit.com/
+location: London, UK
```

Comment:
London is a big place - consider being more precise with your location. We encourage advertisers to say what neighborhood they are in, something like "Islington" would be fine.